app/util/api.js

```diff
@@ -1,33 +1,25 @@
 export function fetchNewsIDsByType(type = 'top') {
     return fetch(`https://hacker-news.firebaseio.com/v0/${type}stories.json?print=pretty`)
         .then((res) => res.json())
-        .then((ids) => {
-            return ids;
-        })
+        .then((ids) => ids);
```

Comment:
no need of round brackets for parameter, but i am ok to keep as it is... upto you..

Reply:
yeah it looks more readable, thanks